crypto/types/compact_bit_array.go

```diff
@@ -47,11 +47,11 @@ func (bA *CompactBitArray) GetIndex(i int) bool {
 		return false
 	}
 
-	return bA.Elems[i>>3]&(uint8(1)<<uint8(7-(i%8))) > 0
+	return bA.Elems[i>>3]&(1<<uint8(7-(i%8))) > 0
 }
 
-// SetIndex sets the bit at index i within the bit array.
-// The behavior is undefined if i >= bA.Count()
+// SetIndex sets the bit at index i within the bit array. Returns true iff the operation
```

Comment:
```suggestion
// SetIndex sets the bit at index i within the bit array. Returns true if the operation
```

Reply:
I used `iff` intentionally there. It means _if and only if_

Reply:
So why not saying `if and only if` instead?

Reply:
It's quiet popular term in CS and it's shorter. If you don't like it then I can change.

Reply:
Yes please, change it.

Reply:
done

Reply:
FYI: https://www.merriam-webster.com/dictionary/iff

Reply:
> FYI: https://www.merriam-webster.com/dictionary/iff

Couldn't give a tally hoot about it. Nerds and amateurs user _iff_ and other acronyms. It's rare to encounter acronyms in professional documentation and manuals. So no iff et similia in developer reference please.